gruntfile.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function (grunt) {
+
+	grunt.initConfig ({
+
+		jshint: {
+			all: {
+				src: ['app/**/*.js']
+			}
+		}
+	});
+
+	grunt.loadNpmTasks('grunt-contrib-jshint');
+	//grunt.loadNpmTasks('grunt-mocha');
+	//grunt.registerTask('default', ['jshint', 'mocha']);
```

Comment:
Ideally include a default task. In this case, if you don't want to use the mocha task, just remove it from the list and make jshint the default task.